src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Touch/PlatformTouchEffect.uwp.cs

```diff
@@ -0,0 +1,207 @@
+﻿using Windows.UI.Xaml.Input;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Platform.UWP;
+using Windows.UI.Xaml.Media.Animation;
+using System;
+using System.Collections.Generic;
+using Xamarin.CommunityToolkit.UWP.Effects;
+using Xamarin.CommunityToolkit.Effects;
+
+[assembly: ExportEffect(typeof(PlatformTouchEffect), nameof(TouchEffect))]
+
+namespace Xamarin.CommunityToolkit.UWP.Effects
+{
+	public class PlatformTouchEffect : PlatformEffect
+	{
+		TouchEffect effect;
+
+		bool isPressed;
+		bool isIntentionalCaptureLoss;
+
+		Storyboard pointerDownStoryboard;
+		Storyboard pointerUpStoryboard;
+
+		protected override void OnAttached()
+		{
+			effect = TouchEffect.PickFrom(Element);
+			if (effect?.IsDisabled ?? true)
+				return;
+
+			effect.Element = Element as VisualElement;
+			if (effect.NativeAnimation)
+			{
+				var nativeControl = Container;
+				if (String.IsNullOrEmpty(nativeControl.Name))
```

Comment:
this is for me, just to make me happy xP
```suggestion
				if (string.IsNullOrEmpty(nativeControl.Name))
```

Reply:
Me too 👍